examples/bri-3/src/bri/workgroup/workgroups/api/workgroups.controller.spec.ts

```diff
@@ -75,22 +76,27 @@ describe('WorkgroupsController', () => {
         DeleteWorkgroupCommandHandler,
         GetWorkgroupByIdQueryHandler,
         WorkgroupStorageAgent,
+        WorkgroupProfile,
       ],
     })
       .overrideProvider(WorkgroupStorageAgent)
-      .useValue(mockWorkgroupStorageAgent)
+      .useValue(mockDeep<WorkgroupStorageAgent>())
       .overrideProvider(BpiSubjectStorageAgent)
       .useValue(mockBpiSubjectStorageAgent)
```

Comment:
is this intentional or miss?

Reply:
could you please clarify?

Reply:
@Manik-Jain  This is ok, bpi subject storage agent is out of scope of this PR